### ISSUE TITLE
Inference in INT8 precision implementation worst than FP32 for storage

### ISSUE THREAD

**Issue by philwg** [NVIDIA-AI-IOT/torch2trt#502]
When I use torch2trt to optimize inference speed on my NVidia AGX Xavier, the state_dicts of TRT_model saved to disk are surprisingly heavy. For example, with a Googlenet Classifier (with custom binary classifier on top), I have :
- original file (.pt) = 22 M
- default conversion mode (FP32) = 50M (??? ... but same accuracy than original !
- FP16 mode = 17M ... good increase of inference speed with very small accuracy decay
- INT8 mode (with calibration dataset method) = 55M (??? ... same speed & same accuracy than default)

It seems nothing is really done in INT8 mode, no ? with an increased need for the storage ...
Can someone make sense of the above numbers?
Is there somewhere a good description of a recommended way to implement the inference based on INT8 TRT optimized (quantiized ?) model from image acquisition to prediction ?

Thanks,
philwg.
